<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Debug</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateKeyButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>generateKey</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>keyText</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="inputText">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>181</width>
      <height>75</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="keyText">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>131</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="debugTextEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>551</width>
      <height>75</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="modeComboBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Encrypt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Encrypt &amp; Decrypt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Decrypt</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="algorithmComboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>DES</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AES</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>algorithm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>mode</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>361</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>651</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <zorder>frame_6</zorder>
   <zorder>frame_4</zorder>
   <zorder>frame_5</zorder>
   <zorder>frame_3</zorder>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>startButton</zorder>
   <zorder>label</zorder>
   <zorder>generateKeyButton</zorder>
   <zorder>label_3</zorder>
   <zorder>inputText</zorder>
   <zorder>keyText</zorder>
   <zorder>label_4</zorder>
   <zorder>debugTextEdit</zorder>
   <zorder>modeComboBox</zorder>
   <zorder>algorithmComboBox</zorder>
   <zorder>label_2</zorder>
   <zorder>label_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEncry_Decry">
    <property name="title">
     <string>En_Decry.</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEuclid">
    <property name="title">
     <string>Euclid</string>
    </property>
    <addaction name="actionEx_Euclid_alg"/>
   </widget>
   <widget class="QMenu" name="menuRSA">
    <property name="title">
     <string>RSA</string>
    </property>
    <addaction name="actionRSA_EnDecry"/>
   </widget>
   <widget class="QMenu" name="menuEIGamal">
    <property name="title">
     <string>EIGamal</string>
    </property>
    <addaction name="actionEIGamal"/>
   </widget>
   <widget class="QMenu" name="menuDiffle_Hellman">
    <property name="title">
     <string>Diffle-Hellman</string>
    </property>
    <addaction name="actionDiffle_Hellman"/>
   </widget>
   <addaction name="menuEncry_Decry"/>
   <addaction name="menuEuclid"/>
   <addaction name="menuRSA"/>
   <addaction name="menuEIGamal"/>
   <addaction name="menuDiffle_Hellman"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEx_Euclid_alg">
   <property name="text">
    <string>Ex_Euclid.alg</string>
   </property>
  </action>
  <action name="actionRSA_EnDecry">
   <property name="text">
    <string>RSA_En&amp;Decry</string>
   </property>
  </action>
  <action name="actionEIGamal">
   <property name="text">
    <string>EIGamal</string>
   </property>
  </action>
  <action name="actionDiffle_Hellman">
   <property name="text">
    <string>Diffle-Hellman</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
